google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/hql/HQLTest.java

```diff
@@ -2073,25 +1963,6 @@ public void test_hql_between_predicate_example_1() {
 		});
 	}
 
-	@Test
-	@RequiresDialect(H2Dialect.class)
-	@RequiresDialect(PostgreSQL81Dialect.class)
-	@RequiresDialect(MySQL5Dialect.class)
-	public void test_hql_between_predicate_example_2() {
-
-		doInJPA( this::entityManagerFactory, entityManager -> {
-			//tag::hql-between-predicate-example[]
-
-			List<Person> persons = entityManager.createQuery(
-				"select p " +
-				"from Person p " +
-				"where p.createdOn between '1999-01-01' and '2001-01-02'", Person.class )
```

Comment:
What does this `between` translate to ?

Reply:
Seems like it's just the the vanilla Between keyword: https://www.w3schools.com/sql/sql_between.asp

It works in Spanner.